passenger-service/src/main/java/com/vlad/kuzhyr/passengerservice/service/impl/PassengerServiceImpl.java

```diff
@@ -53,47 +59,59 @@ public PassengerResponse createPassenger(PassengerRequest passengerRequest) {
         String passengerRequestEmail = passengerRequest.email();
         String passengerRequestPhone = passengerRequest.phone();
 
-        if (passengerRepository.existsPassengerByEmailAndIsEnabledTrue(passengerRequestEmail)) {
-            throw new PassengerAlreadyExistsException(
-                ExceptionMessageConstant.PASSENGER_ALREADY_EXISTS_BY_EMAIL_MESSAGE.formatted(passengerRequestEmail)
-            );
-        }
+        log.debug("Passenger service. Create passenger. Passenger email: {}, passenger phone: {}",
+            passengerRequestEmail,
+            passengerRequestPhone
+        );
 
-        if (passengerRepository.existsPassengerByPhoneAndIsEnabledTrue(passengerRequestPhone)) {
-            throw new PassengerAlreadyExistsException(
-                ExceptionMessageConstant.PASSENGER_ALREADY_EXISTS_BY_PHONE_MESSAGE.formatted(passengerRequestPhone)
-            );
-        }
+        passengerValidator.validatePassengerEmailAndPhone(passengerRequestEmail, passengerRequestPhone);
 
         Passenger newPassenger = passengerMapper.toEntity(passengerRequest);
         Passenger savedPassenger = passengerRepository.save(newPassenger);
+
+        log.info("Passenger service. Created passenger. Passenger id: {}", savedPassenger.getId());
         return passengerMapper.toResponse(savedPassenger);
     }
 
     @Transactional
     @Override
     public PassengerResponse updatePassenger(Long id, PassengerRequest passengerRequest) {
-        Passenger existPassenger = passengerRepository.findPassengerByIdAndIsEnabledTrue(id)
-            .orElseThrow(() -> new PassengerNotFoundException(
-                ExceptionMessageConstant.PASSENGER_NOT_FOUND_MESSAGE.formatted(id)
-            ));
+        Passenger existPassenger = getExistingPassengerById(id);
```

Comment:
При именовании переменных желательно избегать сокращений, в данном случае "exist" выглядит как глагол.
```suggestion
        Passenger existingPassenger = getExistingPassengerById(id);
```